### ISSUE TITLE
Gw crashes when trying to send (click on) an hero build

### ISSUE THREAD

**Issue by KevinDenis** [HasKha/GWToolboxpp#160]

**Comment by KevinDenis** [comment]
Tried with a 6 man team hero build, containing empty hero slots, maybe this is the cause of the crash ?
loading an empty hero team build slot #6 (for 7th player) makes gw crash.

**Comment by M3dinar** [comment]
I reproduced this issue.
In debug GWToolbox crashed at line HeroBuildsWindow:264.
The value of "build.hero_index" is below 0.

It appeared that the default value of "hero_index" from struct "HeroBuild" is not set.
Setting the value to 0 solves the issue in debug version.
In release the issue is not fixed with the fix. If u set explicitly "Choose hero" to "No hero" there is no crash.

I saw another issue when there is gap in the hero list, no further hero are added after the gap.
Example: 
Koss / None / Jora => Only Koss in party formation
